src/main/java/org/openemp/api/user/OpenempUserApiApplication.java

```diff
@@ -17,7 +22,18 @@
 	 * @param args the input arguments
 	 */
 	public static void main(String[] args) {
-        SpringApplication.run(OpenempUserApiApplication.class, args);
-    }
+		SpringApplication.run(OpenempUserApiApplication.class, args);
+	}
+
+	@Bean
+	public WebMvcConfigurer corsConfigurer() {
+		return new WebMvcConfigurer() {
+			@Override
+			public void addCorsMappings(CorsRegistry registry) {
+				registry.addMapping(Constant.AUTH_LOGIN_URL).allowedMethods(HttpMethod.OPTIONS.name(), HttpMethod.POST.name());
+				registry.addMapping(Constant.USER_ENDPOINT).allowedMethods(HttpMethod.OPTIONS.name(), HttpMethod.POST.name());
+			}
+		};
+	}
```

Comment:
Is it possible to move this to `WebSecurityConfig`?

Reply:
Done !